File: 151004/Bosko/lab1/lab1/DTO/NewsRequestTo.cs
using lab1.DTO.Interface;

namespace lab1.DTO
{
    public class NewsRequestTo(int Id, int? UserId, string? Title, string? Content) : IRequestTo
    {
        public int Id { get; set; } = Id;

        public int? UserId { get; set;} = UserId;

        public string? Title { get; set; } = Title;

        public string? Content { get; set; } = Content;
    }
}

File: 151001/Yankova/RV1/src/main/java/by/bsuir/nastassiayankova/Dto/impl/NewsRequestTo.java
package by.bsuir.nastassiayankova.Dto.impl;

import lombok.Getter;
import lombok.Setter;

import java.sql.Date;

@Setter
@Getter
public class NewsRequestTo {
    private long id;
    private long userId;
    private String title;
    private String content;
    private Date created;
    private Date modified;
}
